_episodes/02-dictionaries.md

```diff
@@ -267,6 +277,28 @@ dict_values([12, 18, 24, 54, 87])
 ~~~
 {: .output}
 
+Note that the *dict_keys* and *dict_values* objects are iterable but are not lists. This means that they can be used somewhere like a `for` loop but you can not index them directly. 
+~~~
+d.values()[0]
+~~~
+{: .language-python}
+~~~
+Traceback (most recent call last):
+  File "<stdin>", line 1, in <module>
+TypeError: 'dict_keys' object is not subscriptable
+~~~
+{: .output}
+
+If you want to index keys or values directly, you can convert them to lists with the `list` function.
+~~~
+d.values()[0]
```

Comment:
Could you change this to:
list(d.values())[0]

Reply:
I completely failed to make the point I was trying to make with that omission!